xrdp/xrdp_types.h

```diff
@@ -32,6 +32,14 @@
 #define MAX_NR_CHANNELS 16
 #define MAX_CHANNEL_NAME 16
 
+/* Code values used in 'xrdp_mm->code=' settings */
+#define XVNC_SESSION_CODE 0
+#define XRDP_SESSION_CODE 10
```

Comment:
XRDP in this context means x11rdp. It is deprecated and replaced with xorgxrdp.
I'm not sure x11rdp supports touch events. However, that's fine for the moment.